cookbook/logging/monolog.rst

```diff
@@ -236,6 +236,52 @@ only for a specific handler.
     generate hundreds of log lines. Consider using tools like the `logrotate`_
     Linux command to rotate log files before they become a problem.
 
+    In case you cannot use a dedicated tool for rotating log files, consider using
+    the special ``rotating_file`` handler defined by Monolog. This handler creates
+    a new log file every day and can also remove old files automatically. To use
+    it, just set the ``type`` option of your handler to ``rotating_file``:
+
+    .. configuration-block::
+
+        .. code-block:: yaml
+
+            # app/config/config_dev.yml
+            monolog:
+                handlers:
+                    main:
+                        type:  rotating_file
+                        path:  %kernel.logs_dir%/%kernel.environment%.log
+                        level: debug
```

Comment:
I think we should show `max_files` - maybe commented out, with a comment above/on the line that says it defaults to zero (infinite)